package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@demos-europe/demosplan-ui": "^0.3.17",
     "@demos-europe/dp-consent": "^1.1.2",
-    "@efrane/vuex-json-api": "0.0.38",
+    "@efrane/vuex-json-api": "github:eFrane/vuex-json-api#7d70712dd3fccd2532312ae5dd19b1542e2aadfe",
```

Comment:
Can't we set it to `0.1.0`?

Reply:
no. I had to create a changerequest, which is not merged/published jet: 
https://github.com/eFrane/vuex-json-api/pull/567